File: Code/dotNetBasics/ConsoleApp/ADStack/Components/YoloV8PedestrianDetector.cs
using ConsoleApp.ADStack.Models;

namespace ConsoleApp.ADStack.Components
{
    /// <summary>
    /// Use Yolo v8 for pedestrian detection
    /// </summary>
    internal class YoloV8PedestrianDetector : IComponent
    {
        public void Start()
        {
            Console.WriteLine("[YoloV8PedestrianDetector] Starting.");
        }

        public void Stop()
        {
            Console.WriteLine("[YoloV8PedestrianDetector] Stopping.");
        }
    }
}

File: Code/Fluent.Java/src/adstack/components/YoloV8PedestrianDetector.java
package adstack.components;

import adstack.models.IComponent;

public class YoloV8PedestrianDetector implements IComponent {

	@Override
	public void start() {
        System.out.println("[YoloV8PedestrianDetector] Starting.");
	}

	@Override
	public void stop() {
        System.out.println("[YoloV8PedestrianDetector] Stopping.");
	}

}
